lib/WUI/netif_settings.h

```diff
@@ -2,18 +2,22 @@
 
 #include "lwip/ip_addr.h"
 
-#define LAN_FLAG_ONOFF_POS (1 << 0) // position of ONOFF switch in lan.flag
-#define LAN_FLAG_TYPE_POS  (1 << 1) // position of DHCP/STATIC switch in lan.flag
+#define LAN_FLAG_ONOFF_POS     (1 << 0) // position of ONOFF switch in lan.flag
+#define LAN_FLAG_TYPE_POS      (1 << 1) // position of DHCP/STATIC switch in lan.flag
+#define LAN_FLAG_INTERFACE_POS (1 << 2) // position of WIFI/ETH switch in lan.flag
 
-#define IS_LAN_OFF(flg)    (flg & LAN_FLAG_ONOFF_POS)        // returns true if flag is set to OFF
-#define IS_LAN_ON(flg)     ((flg & LAN_FLAG_ONOFF_POS) == 0) // returns true if flag is set to ON
-#define IS_LAN_STATIC(flg) (flg & LAN_FLAG_TYPE_POS)         // returns true if flag is set to STATIC
-#define IS_LAN_DHCP(flg)   ((flg & LAN_FLAG_TYPE_POS) == 0)  // returns true if flag is set to DHCP
+#define IS_LAN_OFF(flg)           (flg & LAN_FLAG_ONOFF_POS)        // returns true if flag is set to OFF
```

Comment:
please always close argument into the parentheses like ((flg) & LAN_FLAG_ONOFF_POS)